### PR TITLE
Milestone 3: Styling leadership board

### PR DESCRIPTION
**Pull request by AsumVictor**
# Milestone 3: Styling Leadership Board.
In this PR, I have added style to the appearance of the page
 
 ## Type of change
 * [ ] Add style with tailwindcss
 
 # Checklist:
 * [ ]  My code follows the style guidelines of this project
 * [ ] I have performed a self-review of my code
 * [ ] There is no linters error in my code
 * [ ]  My changes generate no new warnings

### CODE REVIEW

File: src/template.html [hunk 1]
```diff
@@ -14,11 +14,11 @@ <h1 class="text-5xl mt-2 mx-2 text-white font-extrabold">Leaderboard</h1>
...
             <h2 class="text-2xl font-bold">Recent Score</h2>
-            <button type="button" class="py-1 px-2 border bg-slate-400 font-bold refresh">Refresh</button>
+            <button type="button" class="refresh border-blue-600 rounded-md bg-blue-800 text-white border px-3 py-1 font-bold self-end ">Refresh</button>
           </div>
-          <table class="table-auto border-2 border-black w-full mt-10">
-            <thead class="border-4 border-black">
-              <tr>
+          <table class="table-auto border-2 border-blue-600 w-full mt-10">
+            <thead class="border-4 border-blue-600">
+              <tr class="bg-blue-600 text-white">
```
Comment (usorfaitheloho) [inline src/template.html:21]: You did really well well your project, please kindly change the placeholder on the second input box to `your score` as shown in the design given as it is good to follow the given framework for your design 😄 

Your design :
leadership board scst

Framework given :
image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @AsumVictor  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * very good UI implementation✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/template.html (hunk 1 above)
Comment (AsumVictor) [reply src/template.html:21]: Thank you very much! I have added the appropriate placeholder for the score input

**Comment by AsumVictor** [comment]
Thank you very much! I have added the appropriate placeholder for the score input

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @AsumVictor, 👋 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Descriptive Pull Request title & summary. ✔️
- Styled the leaderboard project according to the wireframe ✔️
- Good use of Git flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**